R/initialize_project.R

```diff
@@ -63,28 +80,27 @@ initialize_project <- function(folders = c('scripts', 'original_data',
 #' can be helpful when working with existing projects. See `initialize_project()`
 #' for details on adding template files for new organizations.
 #'
-#' @param to.path Destination file path for template files. Typically, root of Rproject directory.
+#' @param to.path Character string. Destination file path for template files. Typically, root of Rproject directory.
+#' @param overwrite Boolean. Overwrite _quarto.yml and style.css files if they already exist? Defaults to FALSE.
 #' @inheritParams initialize_project
 #'
 #' @return Nothing.
 #' @export
 #'
-fetch_quarto_templates = function(to.path, organization = "WDFW"){
+fetch_quarto_templates = function(to.path, organization = "WDFW", overwrite = FALSE){
   supported_organizations = c("WDFW") ## add more as appropriate.
   ## The associated yaml and style files should be added to the `inst` folder with a subfolder
   ## that matches the organization name
 
-  if(!organization %in% supported_organizations){
-    cli::cli_abort("Argument `organization` must be one of the supported organizations: {supported_organizations}.")
-  }
+  rlang::arg_match(organization, supported_organizations)
   yaml.path = system.file(paste0(organization, "/_quarto.yml"), package = "framrsquared")
```

Comment:
consider glue which is used across the rest of the package

```r
glue::glue("{organization}/_quarto.yml")
```

Reply:
Done.